install.sh

```diff
@@ -21,6 +21,31 @@ then
   eval "export PATH=$PATH:$DEST_DIR"
 fi
 
+if [ "$1" != --no-override ]
+then
+echo "Overriding existing git push, pull, and fetch commands to use secure versions"
+echo "
+# override existing git push, pull, fetch to use secure versions
+kevlar_git () {
+  prefix=\"\"
+  cmd=\$1
+  shift
+  if [ \"\$cmd\" == \"push\" ] || [ \"\$cmd\" == \"pull\" ] || [ \"\$cmd\" == \"fetch\" ]; then
+    prefix=\"secure-\"
+  fi
+  \"\`which git\`\" \"\$prefix\$cmd\" \"\$@\"
+}
+alias git='kevlar_git'" >> $HOME/.bashrc
+
+  if [ $? != 0 ]
+  then
+    echo "Could not write git command overrides to ~/.bashrc"
+    echo "git fetch, pull, and push will not automatically use secure versions"
```

Comment:
we should exit with an error if the git aliasing fails, possibly with an explanation that in the meantime the commands are still available in the path by their full names.